src/Sentry/buildTransitive/Sentry.targets

```diff
@@ -125,11 +126,17 @@
       <SentrySetCommitReleaseOptions Condition="'$(SentryOrg)' != ''">$(SentrySetCommitReleaseOptions) --org $(SentryOrg)</SentrySetCommitReleaseOptions>
       <SentrySetCommitReleaseOptions Condition="'$(SentryProject)' != ''">$(SentrySetCommitReleaseOptions) --project $(SentryProject)</SentrySetCommitReleaseOptions>
 
+      <SentryProguardOptions Condition="'$(ProguardUuid)' != ''">$(SentryProguardOptions) --uuid &quot;$(ProguardUuid)&quot;</SentryProguardOptions>
```

Comment:
do we need a main guard that checks to ensure all of the required props are included before we do this?

I wonder if only some of them are, we'll run the command with broken set of arguments and it'll fail. 

Should we let that happen? are some of these arguments optional?

I believe version is no longer neeeed btw, @szokeasaurusrex will know since he was proposing changes to this stuff.